e2e/bang_test.go

```diff
@@ -54,7 +54,6 @@ func TestSuccessfulBangBang(t *testing.T) {
 	result := icmd.RunCmd(cmd)
 
 	result.Assert(t, icmd.Expected{ExitCode: 0})
-
```

Comment:
let's keep this line :) for me it's readable this way.

Reply:
updated